ethereum/permissioning/src/main/java/org/hyperledger/besu/ethereum/permissioning/NodeSmartContractV2PermissioningController.java

```diff
@@ -113,6 +114,23 @@ private Bytes createPayload(final EnodeURL enodeUrl) {
     }
   }
 
+  private String getHost(final EnodeURL enodeUrl) {
```

Comment:
would love to see some unit tests for this method - and it should live with the EnodeURL class (EnodeURLImpl) - can see this being useful elsewhere

Reply:
@macfarla we did some unit tests and moved getHost() to EnodeURL

Reply:
https://stackoverflow.com/questions/25993225/uri-gethost-returns-null-why

Reply:
I differ to @macfarla on this. She can review your PR. Have you reviewed the failing tests? You can possibly ignore the plugins Checksum issue.